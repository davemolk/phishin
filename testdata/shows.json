{"success":true,"total_entries":1759,"total_pages":88,"page":1,"data":[{"id":696,"date":"1990-04-05","duration":8831401,"incomplete":false,"sbd":true,"remastered":false,"tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null}],"tour_id":8,"venue":{"id":339,"slug":"j-j-mccabe-s","name":"J.J. McCabe's","other_names":[],"latitude":40.014986,"longitude":-105.270546,"shows_count":1,"location":"Boulder, CO","updated_at":"2013-03-24T01:17:40Z"},"venue_name":"J.J. McCabe's","taper_notes":"04-05-90 J.J. McCabes, Boulder, CO\r\n\r\nSBD \u003e Cass/0 \u003e DAT \u003e CDR \u003e EAC \u003e SHN\r\nMocking Tree #1\r\n\r\nSeeded by Ben Mohr (bmohr@udel.edu)\r\ncarini.etree.org\r\n\r\n--------------\r\nDisc 1 (73:08)\r\n--------------\r\nSet 1:\r\n1. Possum (6:47)\r\n2. Ya Mar (7:06)\r\n3. David Bowie (11:22)\r\n4. Carolina (2:00)\r\n5. Oh Kee Pa Ceremony \u003e (1:45)\r\n6. Suzy Greenberg (5:19)\r\n7. You Enjoy Myself (12:39)\r\n8. Lizards (10:12)\r\n9. Fire (4:19)\r\nSet 2:\r\n10. Reba (11:39)\r\n\r\n--------------\r\nDisc 2 (73:42)\r\n--------------\r\nSet 2 cont:\r\n1. Uncle Pen (5:13)\r\n2. Jesus Just Left Chicago (8:09)\r\n3. AC/DC Bag (6:22)\r\n4. Donna Lee (3:23)\r\n5. Tweezer (9:59)\r\n6. Fee (5:13)\r\n7. Cavern (4:58)\r\n8. Mike's Song -\u003e (6:22)\r\n9. I Am Hydrogen -\u003e (2:19)\r\n10. Weekapaug Groove (7:34)\r\n11. If I Only Had a Brain (3:09)\r\n12. Contact (6:20)\r\nEncore:\r\n13. Golgi Apparatus (4:41)\r\n\r\nNotes:\r\n--tape flip in YEM at 11:51 (missing couple second transition into vocal jam)\r\n--tape flip between AC/DC Bag and Donna Lee\r\n--tape flip in Weekapaug Groove at 6:27 (no music missing)\r\nFixes:\r\n--fixed click at track boundary of YEM and Lizards\r\n--smoothed out static at 0:08 in I Am Hydrogen","likes_count":5,"tracks":[{"id":14073,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Possum","position":1,"duration":408033,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":1,"slug":"possum","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/073/14073.mp3","waveform_image":"https://phish.in/audio/000/014/073/waveform-14073.png","song_ids":[595],"updated_at":"2023-10-27T22:33:16Z"},{"id":14074,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Ya Mar","position":2,"duration":427024,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"ya-mar","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":17,"name":"Tease","priority":15,"group":"Song Content","color":"#888888","notes":"Theme from Bonanza by Ray Evans and\n Jay Livingston","transcript":null,"starts_at_second":306,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/074/14074.mp3","waveform_image":"https://phish.in/audio/000/014/074/waveform-14074.png","song_ids":[873],"updated_at":"2023-10-27T22:33:16Z"},{"id":14075,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"David Bowie","position":3,"duration":683024,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"david-bowie","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":17,"name":"Tease","priority":15,"group":"Song Content","color":"#888888","notes":"Theme from Bonanza by Ray Evans and\n Jay Livingston","transcript":null,"starts_at_second":16,"ends_at_second":null},{"id":17,"name":"Tease","priority":15,"group":"Song Content","color":"#888888","notes":"Wipe Out by The Surfaris","transcript":null,"starts_at_second":480,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/075/14075.mp3","waveform_image":"https://phish.in/audio/000/014/075/waveform-14075.png","song_ids":[979],"updated_at":"2023-10-27T22:33:16Z"},{"id":14076,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Carolina","position":4,"duration":121025,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"carolina","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":10,"name":"A Cappella","priority":8,"group":"Instrumentation","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/076/14076.mp3","waveform_image":"https://phish.in/audio/000/014/076/waveform-14076.png","song_ids":[136],"updated_at":"2023-10-27T22:33:16Z"},{"id":14077,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"The Oh Kee Pa Ceremony","position":5,"duration":105927,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"the-oh-kee-pa-ceremony","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/077/14077.mp3","waveform_image":"https://phish.in/audio/000/014/077/waveform-14077.png","song_ids":[566],"updated_at":"2023-10-27T22:33:16Z"},{"id":14078,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Suzy Greenberg","position":6,"duration":319112,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"suzy-greenberg","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/078/14078.mp3","waveform_image":"https://phish.in/audio/000/014/078/waveform-14078.png","song_ids":[742],"updated_at":"2023-10-27T22:33:16Z"},{"id":14079,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"You Enjoy Myself","position":7,"duration":760007,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":1,"slug":"you-enjoy-myself","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":17,"name":"Tease","priority":15,"group":"Song Content","color":"#888888","notes":"Flash Light by Parliament","transcript":null,"starts_at_second":480,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/079/14079.mp3","waveform_image":"https://phish.in/audio/000/014/079/waveform-14079.png","song_ids":[879],"updated_at":"2023-10-27T22:33:16Z"},{"id":14080,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"The Lizards","position":8,"duration":612963,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"the-lizards","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/080/14080.mp3","waveform_image":"https://phish.in/audio/000/014/080/waveform-14080.png","song_ids":[458],"updated_at":"2023-10-27T22:33:16Z"},{"id":14081,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Fire","position":9,"duration":260023,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":0,"slug":"fire","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/081/14081.mp3","waveform_image":"https://phish.in/audio/000/014/081/waveform-14081.png","song_ids":[252],"updated_at":"2023-10-27T22:33:16Z"},{"id":14082,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Reba","position":10,"duration":699481,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":1,"slug":"reba","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/082/14082.mp3","waveform_image":"https://phish.in/audio/000/014/082/waveform-14082.png","song_ids":[616],"updated_at":"2023-10-27T22:33:16Z"},{"id":14083,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Uncle Pen","position":11,"duration":314018,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"uncle-pen","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/083/14083.mp3","waveform_image":"https://phish.in/audio/000/014/083/waveform-14083.png","song_ids":[808],"updated_at":"2023-10-27T22:33:16Z"},{"id":14084,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Jesus Just Left Chicago","position":12,"duration":490031,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":1,"slug":"jesus-just-left-chicago","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":15,"name":"Guest","priority":13,"group":"Instrumentation","color":"#888888","notes":"Dan Mosebee on harmonica","transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/084/14084.mp3","waveform_image":"https://phish.in/audio/000/014/084/waveform-14084.png","song_ids":[415],"updated_at":"2023-10-27T22:33:16Z"},{"id":14085,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"AC/DC Bag","position":13,"duration":383033,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"ac-dc-bag","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/085/14085.mp3","waveform_image":"https://phish.in/audio/000/014/085/waveform-14085.png","song_ids":[11],"updated_at":"2023-10-27T22:33:16Z"},{"id":14086,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Donna Lee","position":14,"duration":204016,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"donna-lee","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/086/14086.mp3","waveform_image":"https://phish.in/audio/000/014/086/waveform-14086.png","song_ids":[221],"updated_at":"2023-10-27T22:33:16Z"},{"id":14087,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Tweezer","position":15,"duration":600033,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"tweezer","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":17,"name":"Tease","priority":15,"group":"Song Content","color":"#888888","notes":"Dave's Energy Guide","transcript":null,"starts_at_second":486,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/087/14087.mp3","waveform_image":"https://phish.in/audio/000/014/087/waveform-14087.png","song_ids":[803],"updated_at":"2023-10-27T22:33:16Z"},{"id":14088,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Fee","position":16,"duration":314018,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"fee","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/088/14088.mp3","waveform_image":"https://phish.in/audio/000/014/088/waveform-14088.png","song_ids":[248],"updated_at":"2023-10-27T22:33:16Z"},{"id":14089,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Cavern","position":17,"duration":299024,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"cavern","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":12,"name":"Alt Lyric","priority":10,"group":"Song Content","color":"#888888","notes":"\"...taking turns at *stabbing* her; the brothel wife then grabbed the knife and slashed me on the tongue; I turned the blade back on the bitch and dropped her in the dung...a cushion convector, a *penile collector*...\"","transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/089/14089.mp3","waveform_image":"https://phish.in/audio/000/014/089/waveform-14089.png","song_ids":[142],"updated_at":"2023-10-27T22:33:16Z"},{"id":14090,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Mike's Song","position":18,"duration":383033,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"mikes-song","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/090/14090.mp3","waveform_image":"https://phish.in/audio/000/014/090/waveform-14090.png","song_ids":[505],"updated_at":"2023-10-27T22:33:16Z"},{"id":14091,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"I Am Hydrogen","position":19,"duration":139990,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"i-am-hydrogen","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/091/14091.mp3","waveform_image":"https://phish.in/audio/000/014/091/waveform-14091.png","song_ids":[360],"updated_at":"2023-10-27T22:33:16Z"},{"id":14092,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Weekapaug Groove","position":20,"duration":455027,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"weekapaug-groove","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/092/14092.mp3","waveform_image":"https://phish.in/audio/000/014/092/waveform-14092.png","song_ids":[836],"updated_at":"2023-10-27T22:33:16Z"},{"id":14093,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"If I Only Had a Brain","position":21,"duration":190015,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"if-i-only-had-a-brain","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/093/14093.mp3","waveform_image":"https://phish.in/audio/000/014/093/waveform-14093.png","song_ids":[391],"updated_at":"2023-10-27T22:33:16Z"},{"id":14094,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Contact","position":22,"duration":381022,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":0,"slug":"contact","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/094/14094.mp3","waveform_image":"https://phish.in/audio/000/014/094/waveform-14094.png","song_ids":[168],"updated_at":"2023-10-27T22:33:16Z"},{"id":14095,"show_id":696,"show_date":"1990-04-05","venue_name":"J.J. McCabe's","venue_location":"Boulder, CO","title":"Golgi Apparatus","position":23,"duration":281522,"jam_starts_at_second":null,"set":"E","set_name":"Encore","likes_count":0,"slug":"golgi-apparatus","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/014/095/14095.mp3","waveform_image":"https://phish.in/audio/000/014/095/waveform-14095.png","song_ids":[304],"updated_at":"2023-10-27T22:33:16Z"}],"updated_at":"2018-12-21T08:10:20Z"}]}