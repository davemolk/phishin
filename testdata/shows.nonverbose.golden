Date:       Venue:         Location:    Duration:
1990-04-05  J.J. McCabe's  Boulder, CO  2h 27m

Total Entries: 1759  Total Pages: 88  Result Page: 1
